src/lightning/fabric/accelerators/registry.py

```diff
@@ -112,12 +109,3 @@ def available_accelerators(self) -> List[str]:
 
     def __str__(self) -> str:
         return "Registered Accelerators: {}".format(", ".join(self.available_accelerators()))
-
-
-def call_register_accelerators(registry: _AcceleratorRegistry, base_module: str) -> None:
```

Comment:
this is API breaking change... lets validate we can do it without replacement
cc: @lantiga

Reply:
I put a legacy version in